dashboard-generator/generator.js

```diff
@@ -134,14 +135,18 @@ window.addEventListener('load', function () {
       _dash.splice(plotIndex, 1);
       updatePostData();
       tr.parentElement.removeChild(tr);
+      countDash--;
```

Comment:
Instead of counting the dashboards independently, could you check the length of `_dash`? That will save us the global counter, and tie our conditions directly to the data model.